<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start server</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>160</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mode_choice">
     <item>
      <widget class="QRadioButton" name="server_sync_btn">
       <property name="text">
        <string>Server sync</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="all_sync_btn">
       <property name="text">
        <string>All sync</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="a_t_a_sync_btn">
       <property name="text">
        <string>All to all sync</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="port_l">
       <property name="text">
        <string>port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="port_input"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nickname_l">
       <property name="text">
        <string>nickname</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nickname_input"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
